swagger.yaml

```diff
@@ -380,28 +772,21 @@ paths:
                   type: integer
                   format: integer
                   description: "Amount of ETH to be unstaked, should be in multiples of 32ETH"
+                network:
+                  type: string
+                  enum: ["ethereum"]
+                  description: "Network for which to perform staking request"
       responses:
         202:
           description: "Accepted - request is being processed in the background"
         200:
           description: "Received successful response from Beacon chain for exiting specified amount of ETH. It takes at a minimum 5 epochs (32 minutes) for a validator to exit after initiating a voluntary exit. This number can be much higher depending on how many other validators are queued to exit."
-          content:
-            "application/json":
-              schema:
-                title: ListUnstakeResponse
-                type: array
-                items:
-                  type: "object"
-                  properties:
-                    transaction_hash: 
-                      type: "string"
-                      description: "hash of the signed exit transaction"
         400:
           description: "Specified amount of ETH is invalid."
-        409:
+        428:
           description: "Specified amount of ETH is not yet in a state available to be unstaked. The amount of ETH should be actively staking, not have been slashed and have been active for at least 256 epochs (~27 hours) since it has been activated to be eligible for unstaking."
-  /v1/ethereum/stats:
-    post:
+  /api/v1/stats:
+    get:
       description: "Get rewards and aggregated node statistics for validators hosted by Chorus/Opus"
```

Comment:
@mksh This is the same description as in `/validators` endpoint, should we update it to make the difference between them more clear?